autotime/__init__.py

```diff
@@ -1,9 +1,18 @@
 from __future__ import print_function
 
 import time
+import sys
 
 from IPython.core.magics.execution import _format_time as format_delta
 
+if sys.platform == 'win32':
+    if sys.version_info >= (3, 3):
+        _timer = time.perf_counter
```

Comment:
Looks like `perf_counter` is available on unix systems as well. Why are you only using it on Windows?

```
In [10]: import platform

In [11]: platform.system()
Out[11]: 'Linux'

In [12]: import time

In [13]: time.perf_counter()
Out[13]: 7048.144386648
```

Reply:
The logic here should be conditional only on Python version: `time.perf_counter` if we're `>=3.3`, `time.clock` otherwise